docs/feature_flags.md

```diff
@@ -125,7 +125,61 @@ histogram (albeit via the text format). With this flag enabled, Prometheus will
 still ingest those conventional histograms that do not come with a
 corresponding native histogram. However, if a native histogram is present,
 Prometheus will ignore the corresponding conventional histogram, with the
-notable exception of exemplars, which are always ingested.
+notable exception of exemplars, which are always ingested. To keep the
+conventional histograms as well, enable `scrape_classic_histograms` in the
+scrape job.
+
+_Note about the format of `le` and `quantile` label values:_
+
+In certain situations, the protobuf parsing changes the number formatting of
+the `le` labels of conventional histograms and the `quantile` labels of
+summaries. Typically, this happens if the scraped target is instrumented with
+[client_golang](https://github.com/prometheus/client_golang) provided that
+[promhttp.HandlerOpts.EnableOpenMetrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/promhttp#HandlerOpts)
+is set to false. In such a case, integer label values are represented in the
+text format as such, e.g. quantile="1" or le="2". However, the protobuf parsing
```

Comment:
Let's use backticks for the verbatim parts from the exposition format or PromQL, i.e. `quantile="1"` or `le="2"`.